index.html

```diff
@@ -17,15 +17,23 @@
 <body>
   <header class="toolbar">
     <a class="logo">Omar Gutierrez</a>
-    <nav class="hamburger">
-      <img src="./images/Hamburger.svg" alt="menu">
+    <nav class="nav_toolbar">
+      <img src="./images/Hamburger.svg" alt="menu" class="hamburger" >
+      <ul class="nav-links">
+        <li class="nav-li"><a href="#">About</a></li>
+        <li class="nav-li"><a href="#">Work</a></li>
+        <li class="nav-li"><a href="#">Contact</a></li>
+        <li class="nav-li"><a href="#"><img src="./images/mail_icon.svg" alt="mail" class="mail_icon"></a></li>
+      </ul>
     </nav>
   </header>
   <section class="headline">
-    <h1 class="greeting">Hey There. I'm Omar.</h1>
-    <h3 class="presentation">I'm a Software Developer</h3>
-    <p class="description">I will help you build a product, feature or website. Look through some of my work and
-      experience! If you like what you see and have a project you need coded, don't hesistate to contact me</p>
+    <div class="content">
+      <h1 class="greeting">Hey There. I'm Omar.</h1>
+      <h3 class="presentation">I'm a Software Developer</h3>
+      <p class="description">I will help you build a product, feature or website. Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don't hesistate to contact me</p>
+    </div>
```

Comment:
- [ ] Hello you really did a great job by implemented some required changes from your previouse reviewer but there are still some changes requested from the previouse reviewer that you skip.
- [ ] According to your previous reviewer ,which says [see here](https://github.com/ommarz/Personal_Portfolio/pull/6#discussion_r1233084701) Currently your design for the header and headline section is not corresponding with the provided figma design and its content are not being correctly represented. I will kindly suggest you make the necessary changes as shown with the red lines between your design and the figma design. It is important that your design matches correctly the provided figma template as this shows your attention to details as a frontend developer and this is a good skill to have 👍 ✨ you can align the text h1 and paragraph to fit the original design on figma .see screeshot below to compare 
- [ ] your design 
![Screenshot 2023-06-18 074240](https://github.com/ommarz/Personal_Portfolio/assets/92458236/f7c26db2-0eeb-4270-821f-eff15c266d99)

figma design 
![Screenshot 2023-06-18 074425](https://github.com/ommarz/Personal_Portfolio/assets/92458236/1acced5e-37c0-49d2-99be-fcfcc0c36339)